controller/search_spaces_blackbox_test.go

```diff
@@ -85,7 +85,7 @@ func (rest *TestSearchSpacesREST) TestSpacesSearchOK() {
 		{"with special chars", args{offset("0"), limit(10), "&:\n!#%?*"}, expects{totalCount(0)}},
 		{"with * to list all", args{offset("0"), limit(10), "*"}, expects{totalCountAtLeast(len(idents))}},
 		{"with multi page", args{offset("0"), limit(10), "TEST"}, expects{hasLinks("Next")}},
-		{"with last page", args{offset(strconv.Itoa(len(idents) - 1)), limit(10), "TEST"}, expects{hasNoLinks("Next"), hasLinks("Prev")}},
+		{"with last page", args{offset(strconv.Itoa(len(idents) - 1)), limit(100), "TEST"}, expects{hasNoLinks("Next"), hasLinks("Prev")}},
```

Comment:
yes, I can see it ;) The reason for this change is that, AFAIU, the test is going to request a page of 10 spaces and will expect to link to "next page" in the response. Hélas, with my refactoring on the work items test where I now create custom spaces which do not get cleaned afterwards, the number of spaces goes beyond 11 (Spinal Tap!) and causes this test to fail. Querying 100 spaces was a late-night, easy fix ;)